test/integration/auth_test.go

```diff
@@ -52,7 +52,8 @@ func TestAuthValidateWithBaseAuthURI(t *testing.T) {
 	avoidCachedToken()
 
 	err = execute(validateCommand, "auth", "validate", "--base-auth-uri", "invalid-base-uri")
-	assertError(t, err, "404 Provided Tenant Name is invalid \n")
+	// assertError(t, err, "404 Provided Tenant Name is invalid \n")
```

Comment:
Is this supposed to be commented? If not, lets use a constant for the message, this way if the message changes the test will match it.